File: app/src/main/java/com/juawapps/newsspread/data/db/DatabaseProvider.java
package com.juawapps.newsspread.data.db;

import android.arch.persistence.room.Room;
import android.content.Context;

/**
 * Provides a database service.
 */

public class DatabaseProvider {

    public static AppDatabase getDatabase(Context applicationContext) {
        return Room.databaseBuilder(applicationContext,
                AppDatabase.class, "shout_database")
                .fallbackToDestructiveMigration()
                .build();
    }
}

File: app/src/main/java/com/juawapps/newsspread/data/db/DatabaseProvider.kt
package com.juawapps.newsspread.data.db

import android.arch.persistence.room.Room
import android.content.Context

/**
 * Provides a database service.
 */

object DatabaseProvider {

    fun getDatabase(applicationContext: Context): AppDatabase {
        return Room.databaseBuilder(applicationContext,
                AppDatabase::class.java, "shout_database")
                .fallbackToDestructiveMigration()
                .build()
    }
}
